src/simulation/SimulationManager.java

```diff
@@ -429,8 +440,16 @@ public String getWorkingDirPath() {
     }
 
     private boolean initializeWithModel(File model) {
-        if (AlloyInterface.compile(model.getPath()) == null) {
-            System.out.println("error. Could not parse model.");
+        try {
+            AlloyInterface.compile(model.getPath());
+        } catch (Err e) {
```

Comment:
Can we catch the specific errors here instead of catching a generic alloy error and then breaking it down.